source/lib.ts

```diff
@@ -168,11 +176,23 @@ const parseOptions = (passedOptions: Partial<Options>): Configuration => {
 	const notUndefinedOptions: Partial<Options> =
 		omitUndefinedOptions(passedOptions)
 
-	// Create the validator before even parsing the rest of the options
+	// Create the validator before even parsing the rest of the options.
 	const validations = new Validations(notUndefinedOptions?.validate ?? true)
 
+	// Warn for the deprecated option.
 	validations.onLimitReached(notUndefinedOptions.onLimitReached)
 
+	// The default value for the `standardHeaders` option is `true`, which
```

Comment:
isn't the default false?

Reply:
Oh yes it is, fixed in [428a0dc](https://github.com/express-rate-limit/express-rate-limit/pull/376/commits/428a0dc3eec71375778aebd94f41efbaabfefb2e)